package.json

```diff
@@ -47,6 +47,8 @@
     "url": "^0.11.0",
     "util": "^0.12.4",
     "vue": "^3.0.11",
-    "vue-router": "^4.0.6"
+    "vue-autosuggest": "^2.2.0",
+    "vue-router": "^4.0.6",
+    "vue3-autocomplete": "^1.0.3"
```

Comment:
Both `vue-autosuggest` and `vue3-autocomplete` are added. Only one of them is used right?

Reply:
👍  `vue-autosuggest` is leftover